src/github.com/getlantern/flashlight/app/settings.go

```diff
@@ -49,11 +51,10 @@ func loadSettings(version, revisionDate, buildDate string) *Settings {
 
 // loadSettings loads the initial settings at startup, either from disk or using defaults.
 func loadSettingsFrom(version, revisionDate, buildDate, path string) *Settings {
-
 	log.Debug("Loading settings")
 	// Create default settings that may or may not be overridden from an existing file
 	// on disk.
-	settings = &Settings{
+	set := &Settings{
```

Comment:
Just keep this variable local now to avoid shadowing issues and generally confusing code.